src/follower.py

```diff
@@ -87,21 +94,56 @@ def stop(self):
 
 
     """
-    Turn the wheels towards the last recognized object.
+    Turn the wheels and pans the camera towards the last recognized object.
     """
     def turn(self):
         self.convert_camera_angle()
-        self._fw.turn(self._turn_angle)
+        self.pan_camera()
 
+        if self.STRAIGHT_ANGLE - self.DEADZONE_WHEEL < self._turn_angle:
+            self._fw.turn(self._turn_angle + self.camera_angle_offset) 
+        elif self.STRAIGHT_ANGLE + self.DEADZONE_WHEEL > self._turn_angle:
```

Comment:
Same thing here, shouldn't this be angle - deadzone?

Reply:
changed